front-end/src/pages/leaderboards/leaderboards.tsx

```diff
@@ -121,7 +121,8 @@ export default function Leaderboards() {
           </Popper>
         </Box>
       </FilterBar>
-      {playerTierTable}
+      {seasonId === 'RISEN' && <Typography variant="h5">SELECT A SEASON TO GET STARTED</Typography>}
+      {seasonId !== 'RISEN' && playerTierTable}
```

Comment:
This isnt the most elegant solution, but it was really quick to implement and i created a task in the backlog to make a propper landing page here